deploy-tool/main.go

```diff
@@ -177,13 +178,16 @@ func main() {
 				continue
 			}
 			log.Printf("generating and certifying P256 key %s", name)
-			_, err := generateKeyPair(dataShareProcessorName, name)
+			privKey, err := generateKeyPair(dataShareProcessorName, name)
 			if err != nil {
 				log.Fatalf("%s", err)
 			}
 
-			// TODO(timg) get certificate over the key, insert it here
-			newCertificates[name] = "TODO get certificate"
+			certificate, err := cert.IssueCertificate(name, privKey)
```

Comment:
"name" here is not what we want to issue the certificate for, but we need to think about how to get the right value. I think we would buy some domain `foo.org`, and then use `${manifestWrapper.KubernetesNamespace}.foo.org`. Of course you'll need to rebase on main after #51 lands to get the version of deploy-tool that plumbs that value in.

Reply:
I'll wait on that merge to resolve this issue.